city_scrapers/spiders/il_investment_policy.py

```diff
@@ -0,0 +1,157 @@
+import re
+from collections import defaultdict
+from datetime import date, datetime, time
+from enum import Enum
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlInvestmentPolicySpider(CityScrapersSpider):
+    name = "il_investment_policy"
+    agency = "Illinois Investment Policy Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["www2.illinois.gov"]
+    start_urls = ["https://www2.illinois.gov/sites/iipb/Pages/MeetingInformation.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+        links = self._parse_links(response)
+
+        for item in response.xpath(
+            "//h2[text()='Agendas']/following-sibling::ul"
+            "[not(preceding-sibling::h2[text()='Minutes'])]"
+            "/li/p"
+        ):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location={
+                    "name": "James R. Thompson Center",
+                    "address": "100 W. Randolph St., Room 16-503, Chicago, Illinois"
+                },
+                source=response.url
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            day = date(meeting['start'].year, meeting['start'].month, meeting['start'].day)
+            meeting_type = self._parse_meeting_type(meeting['title'])
+            meeting['links'] = links.get((day, meeting_type), [])
+
+            yield meeting
+
+    @staticmethod
+    def _parse_title(item):
+        """Parse or generate meeting title."""
+        return item.xpath("text()[normalize-space(.)]").get().lstrip(' -').strip()
```

Comment:
Since the titles can sometimes return odd results like "IIPB Board Meeting Agenda" or "COMMITTEE ON SUDAN AND IRAN RESTRICTIONS" even though we know it's constrained to a few values, we should do something similar to what you're doing in `_parse_meeting_type` to pull the title. If there isn't a match for any of the predefined options we can return the existing text